app/services/provider_reporting_service.rb

```diff
@@ -42,8 +42,16 @@ def group_by_count(column)
     closed = @closed_providers.group(column).count
 
     {
-      open: Provider.send(column.to_s.pluralize).map { |key, _value| x = {}; x[key.to_sym] = open[key] || 0; x }.reduce({}, :merge),
-      closed: Provider.send(column.to_s.pluralize).map { |key, _value| x = {}; x[key.to_sym] = closed[key] || 0; x }.reduce({}, :merge),
+      open: Provider.send(column.to_s.pluralize).map { |key, _value|
```

Comment:
I'm surprised Rubocop didn't complain about using `do end` for multi-line blocks. Do we need to add a rule to enforce this?

Reply:
Sure, that can that be for another time...